collect_app/src/androidTest/java/org/odk/collect/android/feature/external/AndroidShortcutsTest.java

```diff
@@ -29,7 +29,6 @@
             .around(rule);
 
     @Test
-    @Ignore("See comment in #pickAndLaunchShortcutForForm")
```

Comment:
I definitely wanted this running while making these changes so managed to find a workaround.